### ISSUE TITLE
DeprecationWarning when importing iris

### ISSUE THREAD

**Issue by shoyer** [SciTools/iris#3021]
I encountered this issue when attempting to get xarray's test suite working with `-W error`:
```
In [1]: import warnings

In [2]: warnings.filterwarnings('always')

In [3]: import iris
/Users/shoyer/miniconda3/envs/xarray-py36/lib/python3.6/site-packages/iris/config.py:94: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = configparser.SafeConfigParser()
/Users/shoyer/miniconda3/envs/xarray-py36/lib/python3.6/importlib/_bootstrap.py:219: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
  return f(*args, **kwds)
[many duplicate lines omitted]
```

I have Iris 2.0.0 installed from conda-forge on Python 3.6.

**Comment by bjlittle** [comment]
Thanks @shoyer

**Comment by pelson** [comment]
Thanks @shoyer.

The config parser one should be fixed in #3025. The ImportError one is more confusing. Looks like the same is true for pandas:

```
>>> import warnings
>>> warnings.filterwarnings('always')
>>> import pandas
/Users/pelson/miniconda/envs/iris-dev-py3/lib/python3.6/importlib/_bootstrap.py:219: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
  return f(*args, **kwds)
```

Is this a cython issue do you think? https://github.com/cython/cython/issues/1720 perhaps...

**Comment by pelson** [comment]
Fixed in ``v2.1.0``. The pandas deprecation remains, but isn't in Iris I think (please correct if wrong).
